api/api-doc/storage_service.json

```diff
@@ -884,7 +884,7 @@
                       },
                       {
                           "name":"scope",
-                          "description":"Stream phase scope",
+                          "description":"Stream phase scope (options: all, dc, rack, node. default: all)",
```

Comment:
might want to the change in this file into 'api: New "scope" API param to load-and-stream calls'.

Reply:
Yikes, thanks!